janitor/tasks/labware_location/main.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import time
+from janitor.tasks.labware_location.sql_queries.sql_queries import GET_LOCATIONS_QUERY, WRITE_TO_LABWARE_LOCATION_QUERY
```

Comment:
They could be defined locally in this file which would get rid of the import line, but this script is primarily for the `sync_changes_from_labwhere` function to be used by APScheduler so I decided to move the setup stuff (loading in queries) to a separate file.

`crawler` does a similar thing but has the SQL queries written out in the Python script instead of in `.sql` files, so I wanted to be consistent with that but with the SQL queries in separate files for better readability.